get_plink_subsets.py

```diff
@@ -31,20 +31,21 @@
 chroms = list(range(1,23))+['X']
 
 
-def get_pops_list(args):
+def get_pops_list(pops: str):
     r'''
     Generates list of population combinations. If `pops`=None, this will read 
-    the phenotype manifest to get all population combinations.
+    the phenotype manifest to get all population combinations. `pops` should be 
+    a string in the format "POP1-POP2-POP3" (for instance, "AFR-EUR-MID")
     '''
-    if args.pops is None:
+    if pops is None:
         pheno_manifest = hl.import_table(get_pheno_manifest_path())
         pops_list_all = pheno_manifest.pops.collect()
-        pops_list_all = sorted(list(set(pops_list_all)))
+        pops_list_all = sorted(set(pops_list_all))
         pops_list_all = [p.split(',') for p in pops_list_all] # list of lists of strings
         idx = range(args.paridx, len(pops_list_all), args.parsplit)
```

Comment:
You've got some `args` stuff in here - might want to make these parameters and document too